src/content-tags-drawer/data/types.mjs

```diff
@@ -60,42 +60,22 @@
 
 /**
  * @typedef {Object} TaxonomyTagData
```

Comment:
Shouldn't this type be shared with the taxonomy list page? Also, it was _totally_ wrong here :p